x-pack/metricbeat/module/sql/query/query_integration_test.go

```diff
@@ -334,11 +339,52 @@ func GetOracleEnvPassword() string {
 	return password
 }
 
+// GetOracleConnectString builds the Oracle connection string with proper format
 func GetOracleConnectString(host string, port string) string {
-	time.Sleep(300 * time.Second)
```

Comment:
See: `waitForOracleConnection` fn

Instead of waiting 5 minutes, we now use exponential backoff. So, as soon as the ports of Oracle are accessible, a standard wait of 1 minute and 30 seconds (90 seconds) has been introduced to give it time to initialize. From testing, it was found out that it does take time even after health checks pass and all.